babel.config.js

```diff
@@ -781,3 +789,37 @@ function pluginInjectNodeReexportsHints({ types: t, template }, { names }) {
     },
   };
 }
+
+/**
+ * @param {import("@babel/core")} pluginAPI
```

Comment:
```suggestion
 * @param {import("@babel/core").PluginAPI}
```

Reply:
Currently our `babel/core` type definitions are in the Microsoft repo, which does not yet have a `PluginAPI`.

`module "C:/Users/user/AppData/Local/Microsoft/TypeScript/4.7/node_modules/@types/babel__core/index"`